src/checkers/inference/VariableAnnotator.java

```diff
@@ -541,7 +541,7 @@ private VariableSlot addPrimaryVariable(AnnotatedTypeMirror atm, final Tree tree
      * the type.
      */
     public VariableSlot addImpliedPrimaryVariable(AnnotatedTypeMirror atm, final AnnotationLocation location) {
-        VariableSlot variable = slotManager.createVariableSlot(location);
+        VariableSlot variable = createVariable(location);
```

Comment:
This method, `createVariable(final AnnotationLocation location)` contains no logic and only calls the `slotManager`. Should we simply remove that method?

Reply:
If we don't merge this PR: #143 then yes. Otherwise, no.